HOW/social/process/how/how-v1.md

```diff
@@ -0,0 +1,163 @@
+---
+HOW: /social/process/how
+Status: Draft
+Created: 2024-11-28
+Version: idx/1
```

Comment:
no Source? Could be the same as HOW here : ) Or did I misunderstand what Source was for?

Reply:
Hah!  Yes there should be a source and it's the self-referential bootstrapping source.  I'll fix that.